css/selectors/is.json

```diff
@@ -220,7 +220,7 @@
                 "version_added": false
               },
               "samsunginternet_android": {
-                "version_added": false
+                "version_added": "15.0"
```

Comment:
The mirroring script appears to have missed the top section of this file. Currently, Samsung Internet isn't marked as supporting :is().